config/locales/idv/en.yml

```diff
@@ -5,6 +5,7 @@ en:
       bad_dob: Your date of birth must be a valid date.
       duplicate_ssn: An account already exists with the information you provided.
       fail: Sorry, we can’t verify your identity based on the information you submitted.
+      finance_number_length: Number must be between 8 and 30 digits.
```

Comment:
Should this be dynamic, based on `VALID_MINIMUM_LENGTH` and `VALID_MAXIMUM_LENGTH`?